examples/bmp388/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers/bmp388"
+)
+
+func main() {
+
+	time.Sleep(time.Second)
```

Comment:
Is there a specific reason for this startup delay in the example?

Reply:
No not really, it's left over from testing, I removed it.